styles/globals.css

```diff
@@ -677,17 +676,6 @@ select {
   background-color: transparent;
 }
 
-.srv-validation-message {
-  color: red;
-  font-size: 1rem;
-  position: absolute;
-  top: 100%;
-}
-.srv-validation-message ~ input,
-.srv-validation-message ~ textarea,
-.srv-validation-message ~ select {
-  border: 1px solid red !important;
-}
```

Comment:
why you removed this? what was this doing?

Reply:
It got removed when I solved merge conflicts and didn't observe this line. 
Its not present in my local codebase also.
<img width="793" alt="Screenshot 2023-07-02 at 11 32 39" src="https://github.com/anmode/grabtern-frontend/assets/91741581/6571a5c8-8d09-4d6b-a9ef-e352b664aada">